### DISCUSSION TITLE
表格字体支持倾斜和下划线吗

### DISCUSSION THREAD

**Discussion by KuduroJS** [antvis/S2#2384] [Q&A] [answered]
如果不支持，怎么实现比较方便呢。
谢谢。

**Comment by lijinke666** [comment, marked as answer]
下划线需要自行在文本下分绘制一条线, 可以参考链接跳转的实现自定义单元格, 在单元格的文字下方绘制,  或者直接使用 S2 内置的 `链接跳转`, 默认就会绘制下划线, https://s2.antv.antgroup.com/examples/interaction/advanced/#pivot-link-jump

https://github.com/antvis/S2/blob/82a67378b0e17383ded27fdd9567752625400f0f/packages/s2-core/src/cell/base-cell.ts#L286-L295


字体倾斜, 配置文本主题的 `fontStyle: "italic"` 即可, 文档有遗漏, 后续补充

![image](https://github.com/antvis/S2/assets/21015895/003f011d-815e-4f19-b354-e9dff8a5f2ce)


![image](https://github.com/antvis/S2/assets/21015895/9033a213-4beb-452a-9408-8f71892a4295)
